spec/services/activitypub/process_status_update_service_spec.rb

```diff
@@ -220,7 +220,10 @@ def poll_option_json(name, votes)
       end
 
       it 'does not update the text, spoiler_text or edited_at' do
-        expect { subject.call(status, json) }.not_to change { s = status.reload; [s.text, s.spoiler_text, s.edited_at] }
+        expect { subject.call(status, json) }.not_to change {
+                                                       s = status.reload
+                                                       [s.text, s.spoiler_text, s.edited_at]
+                                                     }
```

Comment:
```suggestion
        expect do 
          subject.call(status, json)
        end.not_to change do
          s = status.reload
          [s.text, s.spoiler_text, s.edited_at]
        end
```

frankly i think both alternatives look much worse then just allowing the semicolon

Reply:
There is an override that would probably keep it the same https://docs.rubocop.org/rubocop/cops_style.html#allowasexpressionseparator-true

Reply:
I reverted the original cleanup and re-ran with the override, which keeps everything unchanged